### ISSUE TITLE
Automatically add aria-attributes in the post-footer grid

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#6778]
Now that the `post-list` has been removed from the `post-footer`, users of the design system need to manually add ids to the footer grid titles and `aria-labelledby` attributes to the footer grid lists.
To ensure good accessibility and remove potential for manual mistakes, these ids and aria-attributes should be automatically added by the `post-footer` component.

**Comment by myrtasakellariou** [comment]
Closing this ticket since the requirement isn’t possible with the current markup. 

`aria-labelledby` cannot work in this scenario because the label element is inside the Shadow DOM while the list is projected via a slot. The only option would be to set the id at the host level, but this is not applicable in this case.
